packages/plugin-dom-layout/src/DomReconciliationEngine.ts

```diff
@@ -1060,7 +1103,15 @@ export class DomReconciliationEngine {
         if (old) {
             oldIdsToRelease.push(old.id);
             if (typeof old.object.detach === 'function') {
-                old.object.detach(...old.dom);
+                try {
+                    old.object.detach(...old.dom);
+                } catch (e) {
+                    console.warn(
+                        'Can not detach the nodes from a rendering object, one of the references was may be wrong.',
```

Comment:
idem detach

Reply:
An error occured while detaching. The target rendering might already have been deleted. Multiple references might target this rendering, in which case at least one of them is wrong.